tests/integration/api_repo_file_create_test.go

```diff
@@ -50,95 +49,7 @@ func getCreateFileOptions() api.CreateFileOptions {
 	}
 }
 
-func getExpectedFileResponseForCreate(repoFullName, commitID, treePath, latestCommitSHA string) *api.FileResponse {
```

Comment:
Why delete these functions and tests?

Reply:
Duplicate code and testing. Same function and test are in `api_repo_file_create_or_update_test.go`